### PR TITLE
Add gemini

### PR DESCRIPTION
**Pull request by taturl**
Google just released gemini-2.0-flash and seems like it's pretty cool and inexpensive (rate-limited free but only "preview"). 

So let's expand the list of our LLM providers.

This PR adds just the required libraries, API token and a way to store the client in-memory.

In next PR, we could change the current /gpt to use this. I think we don't even need a list of API's in the end, just one is enough for users (aka latest and greatest).

### CODE REVIEW

File: docker-compose.ci.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ services:
       BOT_TOKEN: ${BOT_TOKEN}
       DJANGO_SECRET_KEY: ${DJANGO_SECRET_KEY}
       OPEN_WEATHER_API_KEY: ${OPEN_WEATHER_API_KEY}
-      OPENAI_API_KEY: ${OPENAI_API_KE}
+      OPENAI_API_KEY: ${OPENAI_API_KEY}
```
Comment (taturl) [inline docker-compose.ci.yml:16]: What is this typo btw

Comment (Latemus) [reply docker-compose.ci.yml:16]: It's yours :) Now that I think of it, I suppose this `docker-compose.ci.yml` should not need these variables exposed as this is only used by github actions and the tests shouldn't rely on having these variables anyway.

**Review by Latemus — APPROVED** [review]

File: bobweb/bob/google_genai_api_utils.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        self.default_client = None
+    
+    def setup_client(self):
+        """
+        Initializes a client if api key is set as environment variable.
+        """
+        api_key = os.getenv('GOOGLE_GENAI_API_KEY')
+        if api_key is None or api_key == '':
+            logger.error('GOOGLE_GENAI_API_KEY is not set. No response was generated.')
+            raise ResponseGenerationException('Google Gen AI API-avain puuttuu ympäristömuuttujista')
```
[the remark this answers is not in the document]
Comment (taturl) [reply bobweb/bob/google_genai_api_utils.py]: Changed to English.

File: docker-compose.ci.yml (hunk 1 above)
Comment (taturl) [reply docker-compose.ci.yml:16]: Maybe we can remove all these in later PR.

**Pull request merged**